mkosi/machine.py

```diff
@@ -32,15 +36,27 @@
 
 class Machine:
     def __init__(self, args: Optional[Sequence[str]] = None, debug: bool = False) -> None:
-        # Remains None until image is built and booted, then receives pexpect process
+        # Remains None until image is built and booted, then receives pexpect process.
         self._serial: Optional[pexpect.spawn] = None
         self.exit_code: int = -1
         self.debug = debug
         self.stack = contextlib.ExitStack()
         self.args: MkosiArgs
 
-        # We make sure to add the arguments in the machine class itself, rather than typing this for every testing function.
         tmp = parse_args(args)["default"]
+
+        # We could have the Verb passed by an environment variable, rather than only args.
+        if tmp.verb not in (Verb.qemu, Verb.boot):
+            default_verb = os.environ.get("MKOSI_TEST_DEFAULT_VERB")
+            # This way, if environment var is not set, we assign nspawn by default to test cases with no Verb
+            if default_verb in (Verb.boot.name, None):
+                tmp.verb = Verb.boot
+            elif default_verb == Verb.qemu.name:
+                tmp.verb = Verb.qemu
+            else:
+                die("No valid verb was entered.")
```

Comment:
The problem with this is: if someone gives us a verb like `shell`, `help`, we'll override that value with `boot` or `qemu`, instead of calling `die`.
So I should change to something like `if tmp.verb == None`